src/sentry/static/sentry/app/components/dropdownLink.jsx

```diff
@@ -2,34 +2,20 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 
+import DropdownMenu from './dropdownMenu';
+
 class DropdownLink extends React.Component {
   static propTypes = {
+    ...DropdownMenu.propTypes,
```

Comment:
What do you think about explicitly listing the props here? It's not ideal but might make this easier to read and fix the storybook issue. Also, it might not be necessary to expose the whole list of props on `DropdownMenu` here e.g. the `onClickOutside` method

Reply:
I think this is fine for now - we could open up an issue for storybook to add support (I haven't looked but it might even already exist). Since we don't use any of `<DropdownMenu>`'s props, I don't think we should explicitly declare them here.